app/template.ejs

```diff
@@ -7,7 +7,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta name="description" content="Accounts: Convention on Biological Diversity">
         <!-- Slaask. need to be placed just before require.JS script tag -->
-    <script src="https://cdn.slaask.com/chat.js"></script>
+    <script src="https://cdn.slaask.com/chat.js?v=<%=gitVersion%>"></script>
```

Comment:
why reload slaask on new accounts versions?

Reply:
My thought was if it was updated (version# in bower), the user would still get the cached version. Am I incorrect?

Reply:
we do not load it from our server, its loaded form their server